servers/physics_2d/space_2d_sw.cpp

```diff
@@ -34,8 +34,8 @@
 #include "core/os/os.h"
 #include "core/pair.h"
 #include "physics_server_2d_sw.h"
-_FORCE_INLINE_ static bool _can_collide_with(CollisionObject2DSW *p_object, uint32_t p_collision_mask, bool p_collide_with_bodies, bool p_collide_with_areas) {
-	if (!(p_object->get_collision_layer() & p_collision_mask)) {
```

Comment:
If we're updating this, then we should add the missing blank line at the end of the includes.